src/Turn.js

```diff
@@ -1,32 +1,34 @@
 class Turn {
-    constructor(userGuess, currentCard) {
-        this.userGuess = userGuess,
-        this.currentCard = currentCard,
-        this.guessCorrect = null;
+  constructor(userGuess, currentCard) {
+    this.userGuess = userGuess,
+    this.currentCard = currentCard,
+    this.guessCorrect = null;
+  }
+  returnGuess() {
+    return this.userGuess;
+  }
+  returnCard() {
+    return this.currentCard;
+  }
+  evaluateGuess() {
+    if (this.userGuess === this.currentCard.correctAnswer) {
+      this.guessCorrect = true;
+      return true;
+    } else {
+      this.guessCorrect = false;
+      return false;
     }
-    returnGuess() {
-        return this.userGuess;
-    }
-    returnCard() {
-        return this.currentCard;
-    }
-    evaluateGuess() {
-        if (this.userGuess === this.currentCard.correctAnswer) {
-            this.guessCorrect = true;
-            return true;
-        } else {
-            this.guessCorrect = false;
-            return false;
-        }
-        //this.userGuess === this.currentCard.correctAnswer ? true : false;
-    }
-    giveFeedback() {
-        if (this.guessCorrect === true) {
-            return "correct!";
-        } else {
-            return "incorrect!";
-        }
+  }
+  giveFeedback() {
+    switch(this.guessCorrect) {
+      case true:
+				return "correct!"
+				break;
+			case false:  
+				return "incorrect!";
+				break;
```

Comment:
Weird spacing here.  What is the decision behind when to use if/else vs ternary vs switch/case?  Is it part of the spec, or for practice, or a more specific reason?

Reply:
Switch is more practice than anything. Trying to practice using ternaries to see where and when I can use them. Clean code is pretty.